googleMapsApiModule.py

```diff
@@ -20,7 +20,15 @@ def direction(origin,destination):
     logging.debug(result_url)
     webbrowser.open_new(result_url)
 
+def timezone(search_location,timestamp):
+    address = search_location.lower()
+    address = address.replace(" ","+")
+    result_url = f'https://maps.googleapis.com/maps/api/timezone/json?location={address}&timestamp={timestamp}&key={config.key}
+    print(result_url)
+    webbrowser.open_new(result_url)
+
 @logger_config.logger 
+
```

Comment:
Does this new line not affect the logging for sure?

Reply:
I don't know I am testing it, MySQL is throwing issues, I will check all the code locally then I will make a final commit.

Reply:
It does, please undo this addition of extra line